apps/web/src/features/walletconnect/components/WcProposalForm/styles.module.css

```diff
@@ -10,7 +10,24 @@
 }
 
 .chain {
-  margin: 2px;
+  margin: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.chainContainer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left-align;
```

Comment:
```suggestion
  justify-content: flex-start;
```

Please test that it works.

Reply:
<img width="497" alt="Screenshot 2025-04-15 at 12 00 11" src="https://github.com/user-attachments/assets/1b33e4e6-461a-433e-8493-553903d27d5b" />

This is the result with flex-start.